common/cloudfront.js

```diff
@@ -29,7 +29,14 @@ function makeBehaviourItem({
         ? ['HEAD', 'DELETE', 'POST', 'GET', 'OPTIONS', 'PUT', 'PATCH']
         : ['HEAD', 'GET'];
 
-    const globalQuerystrings = ['draft', 'version', 'social'];
+    const globalQuerystrings = [
```

Comment:
We can presumably remove `draft` and `version` now, or do they still exist?

Reply:
Oh good call – yeah, let's do that.